packages/packages/top-bar/src/components/top-bar.tsx

```diff
@@ -1,23 +1,53 @@
-import React from 'react';
+import React, { useLayoutEffect } from 'react';
+import { AppBar, Grid, Box, IconButton, styled } from '@elementor/ui';
+import ElementorIcon from './icons/ElementorIcon';
+import PlusIcon from './icons/PlusIcon';
+
+interface ExtendedWindow extends Window {
+    $e: {
+        route: ( route: string ) => void;
+    };
+}
```

Comment:
why not declare global?

Reply:
Was replaced with `// @ts-ignore` because the command is a temp solution that will be replaced soon.